zeplin/Shared/NetworkService.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  NetworkService.swift
+//  zeplin
+//
+//  Created by yagiz on 3/29/20.
+//  Copyright © 2020 Yagiz Nizipli. All rights reserved.
+//
+
+import Foundation
+import Moya
+import KeychainSwift
+
+enum ZeplinService {
+    case getCurrentUser
+    case getProjects
+    case getScreens(project: Project)
+    case deleteScreen(screen: Screen, project: Project)
+}
+
+
+extension ZeplinService: TargetType {
+    var baseURL: URL { URL(string: "https://api.relevantfruit.com/v1/zeplin")! }
```

Comment:
`v1/zeplin/` kısmını endpoint içine al. o v1'in anlamı endpoint'i versiyonlamak değil mi zaten?
Ayrıca `zeplin/v1/` daha doğru sanki eğer aynı base api üzerinden farklı uygulamaları beslemek istiyorsan.